README.md

```diff
@@ -277,7 +321,7 @@ What follows is a deeper dive into both of these content types.
 
 For any kind of repeating content, this is the basic process:
 
-1. In the vars file for the section you're in (e.g., `blog/_vars-blog.html`),
+1. In the vars file for the section you're in (e.g., `newsroom/_vars-newsroom.html`),
```

Comment:
Newsroom landing is now in wagtail, the only bit that isn't is the Press Resources (thanks @schaferjh for the clarification)